dash/backend/src/modules/settings/services/app-settings.service.ts

```diff
@@ -16,25 +14,6 @@ export class AppSettingsService {
         return plainToInstance(AppSettingsDto, name ? appSettings.pop() : appSettings, {enableImplicitConversion: true, excludeExtraneousValues: true});
     }
 
-    async getLicenseSettings(): Promise<any> {
-        return await this.appSettingsDao.getAppSettings(AppSettingsType.LICENSE_SETTINGS);
-    }
-
-    async doesLicenseHaveImageScanningEnforcement(): Promise<boolean> {
-        return this.getLicenseSettings()
-          .then((licenseFeatures) => {
-              // Returns true if there are license features, and image scanning enforcement is an enabled feature
-              return !!licenseFeatures &&
-                licenseFeatures.some(feature => feature.name === LicenseFeatures.IMAGE_SCANNING_ENFORCEMENT && feature.value === 'true');
-          });
-
-    }
-
-    async getAllSettings(type: AppSettingsType): Promise<AppSettingsDto[]> {
```

Comment:
Was this not being used?

Reply:
The exclusive usage was in the getLicenseAndInstanceKeys() removed below

Reply:
Further context, this was a redundant function with the getSettings() function above